src/main/java/com/ebay/myriad/executor/MyriadExecutor.java

```diff
@@ -4,14 +4,7 @@
 import org.apache.mesos.Executor;
 import org.apache.mesos.ExecutorDriver;
 import org.apache.mesos.MesosExecutorDriver;
-import org.apache.mesos.Protos.ExecutorInfo;
-import org.apache.mesos.Protos.FrameworkInfo;
-import org.apache.mesos.Protos.SlaveInfo;
-import org.apache.mesos.Protos.Status;
-import org.apache.mesos.Protos.TaskID;
-import org.apache.mesos.Protos.TaskInfo;
-import org.apache.mesos.Protos.TaskState;
-import org.apache.mesos.Protos.TaskStatus;
+import org.apache.mesos.Protos.*;
```

Comment:
I tend to follow the school of disallowing .* imports, to reduce the ambiguity caused by multiple identically named classes. (Of course, I'd be willing to back down if others strongly disagree.)
http://www.coderanch.com/how-to/java/WildCardVsSpecificImports
https://google-styleguide.googlecode.com/svn/trunk/javaguide.html#s3.3.1-wildcard-imports